mssb_1099b_to_txf.py

```diff
@@ -47,16 +47,30 @@
         r'(?P<proceeds>\$[0-9,.]+)\s+'
         r'(?P<cost>\$[0-9,.]+)\s', re.DOTALL|re.MULTILINE)
 
+
+def formatShareQuantity(quantity):
```

Comment:
What do you think about converting to a float and using is_integer? That might be simpler and more robust.

Reply:
That will work for integers, but it's still possible to have floating point numbers like `123.45000` (this is how it would be formatted on my 1099B). It would be nice to trim the zeroes in that case as well.

I considered something like `str(float('123.45000'))`. This seems to work for the handful of numbers I tried, however I know that some fractional numbers can't be represented accurately as a double. I actually haven't found a case where the conversion gives an inaccurate answer, but it seemed like an unnecessary risk.

Reply:
I was thinking

```
>>> str(float('123.45000'))
'123.45'
```

and you also have a good point about rounding errors. To keep as a string, what do you think about a simpler `.rstrip('0').rstrip('.')` ?

I also searched about this and found what is most precise answer (https://docs.python.org/3.9/library/decimal.html#decimal-faq):

```
>>> d = decimal.Decimal('123.45000')
>>> d = d.quantize(decimal.Decimal(1)) if d == d.to_integral() else d.normalize()
>>> '{:f}'.format(d)
'123.45'
```

Reply:
PTAL. I updated this to use your `rstrip()` idea.